app/src/components/container/MapContainer.jsx

```diff
@@ -13,11 +13,29 @@ export default class MapContainer extends Component {
   handleFullScreenChange() {}
 
   componentDidMount() {
-    let map = new AstroMap("map-container", "DIONE", {});
+    let map = new AstroMap("map-container", this.props.target, {});
     new Projection().addTo(map);
     new MousePosition({
       numDigits: 2,
-      targetPlanet: "mars"
+      targetPlanet: this.props.target
+    }).addTo(map);
+
+    map.on("fullscreenchange", this.handleFullScreenChange());
+  }
+
+  componentDidUpdate() {
+    let oldContainer = document.getElementById("map-container");
+    let parent = oldContainer.parentNode;
+    let newContainer = document.createElement("div");
+    parent.removeChild(oldContainer);
+    newContainer.setAttribute("id", "map-container");
+    parent.appendChild(newContainer);
+
+    let map = new AstroMap("map-container", this.props.target, {});
+    new Projection().addTo(map);
+    new MousePosition({
```

Comment:
I am not sure what happened to master, but we were adding a scale control to the map. It looks like it got deleted somehow, not by your PR. Could you add in adding the control? Should be: new L.Control.Scale({ imperial: false }).addTo(map);